src/main/java/medicaldoctor/controllers/LogOutController.java

```diff
@@ -26,6 +26,7 @@ void exitButtonClicked(ActionEvent event){
     @FXML
     void loginPageButtonClicked(ActionEvent event) throws Exception{
         AppSession.CONTROLLER_MANAGER.showScreen(LookUp.LOGIN_SCREEN);
+        //This does not logout user in backend.
```

Comment:
```suggestion
        AppSession.setActiveUser(null);
```

Reply:
preferrably would create a logout method on the appsession that will do that plus clear the other fields such as the current visit and current patient